Replace "true"/"false" strings with bool values

gaspar-chilingarov opened the pull request:
Fix the remainders of the Terraform v0.12.x syntax

comment-bot-dev commented:
Thanks for the PR! 🚀
Unfortunately it looks like some of our CI checks failed. See the [Contributing Guide](../blob/master/CONTRIBUTING.md) for details.
- ⚠️check_documentation
 The documentation needs to be regenerated. Please run `make generate_docs`. 
 ```bash 
Checking for documentation generation
diff -r '--exclude=.terraform' '--exclude=.kitchen' '--exclude=.git' '--exclude=autogen' '--exclude=*.tfvars' /workspace/README.md /tmp/tmp.ZwwR8l3AjN/generate_docs/workspace/README.md
129c129
< | disable\_services\_on\_destroy | Whether project services will be disabled when the resources are destroyed | `string` | `true` | no |
---
> | disable\_services\_on\_destroy | Whether project services will be disabled when the resources are destroyed | `bool` | `true` | no |
diff -r '--exclude=.terraform' '--exclude=.kitchen' '--exclude=.git' '--exclude=autogen' '--exclude=*.tfvars' /workspace/modules/gsuite_enabled/README.md /tmp/tmp.ZwwR8l3AjN/generate_docs/workspace/modules/gsuite_enabled/README.md
63c63
< | auto\_create\_network | Create the default network | `string` | `false` | no |
---
> | auto\_create\_network | Create the default network | `bool` | `false` | no |
77,78c77,78
< | disable\_dependent\_services | Whether services that are enabled and which depend on this service should also be disabled when this service is destroyed. | `string` | `true` | no |
< | disable\_services\_on\_destroy | Whether project services will be disabled when the resources are destroyed | `string` | `true` | no |
---
> | disable\_dependent\_services | Whether services that are enabled and which depend on this service should also be disabled when this service is destroyed. | `bool` | `true` | no |
> | disable\_services\_on\_destroy | Whether project services will be disabled when the resources are destroyed | `bool` | `true` | no |
87c87
< | lien | Add a lien on the project to prevent accidental deletion | `string` | `false` | no |
---
> | lien | Add a lien on the project to prevent accidental deletion | `bool` | `false` | no |
91c91
< | random\_project\_id | Adds a suffix of 4 random characters to the `project_id` | `string` | `false` | no |
---
> | random\_project\_id | Adds a suffix of 4 random characters to the `project_id` | `bool` | `false` | no |
diff -r '--exclude=.terraform' '--exclude=.kitchen' '--exclude=.git' '--exclude=autogen' '--exclude=*.tfvars' /workspace/modules/project_services/README.md /tmp/tmp.ZwwR8l3AjN/generate_docs/workspace/modules/project_services/README.md
44,45c44,45
< | disable\_dependent\_services | Whether services that are enabled and which depend on this service should also be disabled when this service is destroyed. https://www.terraform.io/docs/providers/google/r/google_project_service.html#disable_dependent_services | `string` | `true` | no |
< | disable\_services\_on\_destroy | Whether project services will be disabled when the resources are destroyed. https://www.terraform.io/docs/providers/google/r/google_project_service.html#disable_on_destroy | `string` | `true` | no |
---
> | disable\_dependent\_services | Whether services that are enabled and which depend on this service should also be disabled when this service is destroyed. https://www.terraform.io/docs/providers/google/r/google_project_service.html#disable_dependent_services | `bool` | `true` | no |
> | disable\_services\_on\_destroy | Whether project services will be disabled when the resources are destroyed. https://www.terraform.io/docs/providers/google/r/google_project_service.html#disable_on_destroy | `bool` | `true` | no |
Error: Documentation generation has not been run, please run the
'make docker_generate_docs' command and commit the above changes.
 ``` 
- ⚠️check_whitespace
 Failed whitespace check. More details below. 
 ```bash 
Checking for trailing whitespace
./modules/gsuite_enabled/variables.tf:179:  type        = bool 
Error: Trailing whitespace found in the lines above.
Checking for missing newline at end of file
 ``` 
- ⚠️check_terraform
 Failed Terraform check. More details below. 
 ```bash 
Running terraform fmt
modules/gsuite_enabled/variables.tf
--- old/modules/gsuite_enabled/variables.tf
+++ new/modules/gsuite_enabled/variables.tf
@@ -176,7 +176,7 @@
 variable "disable_dependent_services" {
   description = "Whether services that are enabled and which depend on this service should also be disabled when this service is destroyed."
   default     = true
-  type        = bool 
+  type        = bool
 }
 
 variable "enable_shared_vpc_service_project" {
Error: terraform fmt failed with exit code 3
Check the output for diffs and correct using terraform fmt <dir>
 ```

morgante commented on docs/upgrading_to_project_factory_v1.0.md:
```diff
@@ -34,7 +34,7 @@ provider "gsuite" {
 module "project-factory" {
```
We shouldn't touch the old upgrading docs as we won't be backporting this change.

gaspar-chilingarov replied:
ah in that way, sure, will roll it back :)

morgante commented:
@gaspar-chilingarov Thanks for working on this! We've had the old syntax hanging around for too long.

gaspar-chilingarov commented:
@morgante I have no idea why integration tests were failing before :P now it seems to be working. 

I've rebased, squashed, it should be nice single commit now. I'm not sure if this should wait until next major release, because it may accidentaly break somehing somewhere.

morgante commented:
@gaspar-chilingarov Looks like they're passing now, thanks! This shouldn't break anything so I'm going to merge.
